docs/source/scancode-toolkit/cli-reference/core-options.rst

```diff
@@ -0,0 +1,282 @@
+`Core` Options
+==============
+
+.. include::  /scancode-toolkit/rst_snippets/basic_options.rst
+
+.. _cli_core:
+
+.. include::  /scancode-toolkit/rst_snippets/core_options.rst
+
+``--generated`` Options
```

Comment:
That's great doc, but as I am reading this I wonder if this should not be included in the `--info` option rather than be a separate option. (this is just a remark... but your opinion is welcomed as a future evolution of the scancode CLI user experience)

Reply:
I do not follow you here. Are you talking about `--generated` Options? Or are you talking about the rst_snippets include?

Reply:
This is an interesting point. Most other options are in some way dependent on other options, (You've also asked -> Should not `--max-email`  be side-by-side with the `-e` option?) like the `--summary` option is dependent on `--info` and has some dependent options of its own. Rather than "including" them in each other, I think they should be kept in separate sections as it is presently. What can be done to better show these dependencies is I could add Notes on every option who they are dependent on and what other options are dependent.

Reply:
@pombredanne it probably should be included in the `--info`, but this is a scancode-toolkit specific issue I think.

Reply:
Another question, looks like `--generated` is independent of `--info`, and even without `-i` it works normally without errors. Should this dependency be pointed out? 
In other cases, I'm going with notes like : 

> The option --max-email is a sub-option of and requires the option(s) --email